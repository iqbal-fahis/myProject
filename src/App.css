
.App-header {
  font-family:Arial, Helvetica, sans-serif ;
  font-style: italic;
  font-stretch: narrower;
  min-height: 10vh;
  display:flex;
  flex-direction: column;
  align-items: center;
  font-size: calc(10px + 4vmin);
  color: rgb(165, 16, 16);
}
img{
  height: 300px;
}
.Login h1{ color: #FFF;} 
    .Login p{ color: burlywood }
    .Login2 h1{ color: #FFF;} 
    .Login2 p{ color: burlywood }
.error{
      color:white;
      font-style: oblique;
      font-weight: bold;
    }
.Login {
  width: 505px;
  margin:auto;
  border:1px #CCC solid;
  padding:30px;
  background-color: #2285cc;
  color:#FFF;
  height:300px;
}
.field {
  background: #29363f;
  margin:5px 25px;
  width:216px;
  color:#FFF;
  }
  .field {
    border:1px #525d65 solid;
    padding:10px;
    }
.Login2 {
  width: 505px;
  margin:auto;
  border:1px #CCC solid;
  padding:30px;
  background-color: #2285cc;
  color:#FFF;
  height:400px;
  }

  .field2 {
    border:0px #525d65 solid;
    padding:3px;
    }
.field2 {
    background: #29363f;
    margin:5px 25px;
    width:150px;
    color:#FFF;  
}
.lgn_left, .lgn_right { height: 100%; }
    .lgn_left {
    width: 200px;
    border-right:1px #525d65 solid;
    float:left;
    }
    .lgn_right {
    width: 300px;
    float: right;
    text-align:left;
    text-align: right;

    }
.create_btn{
  color: white;
  background-color: darkblue;
  font-size: 10px;
  width: 50px;
  margin-right: 5px;
  padding-right: 5px;
}

  .lgn_btn{
    color: black    ;
    background-color: lightblue;
    size: 0em;
    width: initial;
    margin-right: 50px;
}
.lgn_btn2{
  color: white;
  background-color:lightblue;
  size: 0em;
  width: initial;
  margin-right: 50px;
}
.link{
  text-align: left;
  float: left;
 font-style: oblique;
 font-size: 20px;
 text-indent: 5px;
}
.errorMessage{
  font-style: italic;
  color:greenyellow;
  font-weight: bold;
  font-size: 10px;
}

.hmeLink{
  font-size: x-large;
  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
  font-style: italic;
}
/*.One-para{
  font-size: medium;
  align-items: center;
  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
  color: blue;
}*/


