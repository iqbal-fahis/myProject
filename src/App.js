import React from 'react';
//import ReactDOM from 'react';
import {Component} from 'react';
import {BrowserRouter, Route, Redirect, Switch, Link} from 'react-router-dom';
//import Routes from './routes';
//import axios from "axios";
import './App.css';
import store,{COUNTER_INCREMENT, COUNTER_DECREMENT} from './Store'
import { render } from '@testing-library/react';//
import One from './One'
import Signup from './Signup'
import Welcome from './Welcome'
import Dashboard from './Dashboard'
import Home from './Home'
import UserFirst from './UserFirst';
import Myprofile from './Myprofile';
import Ticket from './Ticket';
import Problem from './Problem';
import Dropdown from './Dropdown';
import Arith from './Arith';
import Add from './Add';
import LifeCycles from './LifeCycles';
import Binding from './Binding';
import ChildOne from './ChildOne';
import Parent from './Parent';
import ChildTwo from './ChildTwo';
class App extends Component{
constructor(props)
{
  super(props);
 
  this.state ={
    name : "",
    loggedInStatus: "NOT_LOGGED_IN",
    user : {}
  }
}

/*checkLoginStatus()
{
  <div><img src="/images/comm.jpg" alt=""/></div>
  axios
  .get("http://localhost:3000/logged_in", { withCredentials: true })
  .then(response => {
     console.log("logged in?", response);
  })
    .catch(error => {
    console.log("check error", error);
  })
}

componentDidMount(){
  this.checkLoginStatus();
}*/


increment =()=>{
  store.dispatch({type: COUNTER_INCREMENT});
  console.log(store.getState().count);
}
decrement =()=>{
  store.dispatch({type: COUNTER_DECREMENT});
  console.log(store.getState().count);
}
render()
{

  return (
  
  <div>
    <p className="App-header">{this.state.name}</p>
    <p className="App-header">This is Reacttt</p>
    <center>
    <button onClick={this.increment}> Increment</button>&nbsp;&nbsp;
    <button onClick={this.decrement}> Decrement</button>
    </center>
    <BrowserRouter >
<Switch>
<Route exact path="/One" component={One}/>
<Redirect from='/One/' to="/Welcome/" />
<Route path="/Welcome" component={Welcome}/>
<Route path="/Signup" component={Signup}/>
<Route path="/Dashboard" component={Dashboard}/>
<Route path="/Home" component={Home}/>
<Route path="/UserFirst" component={UserFirst}/>
<Route path="/Myprofile" component={Myprofile}/>
<Route path="/Ticket" component={Ticket}/>
<Route path="/Problem" component={Problem}/>
<Route path="/Dropdown" component={Dropdown}/>
<Route path="/Arith" component={Arith}/>
<Route path="/Add" component={Add}/>
<Route path="/LifeCycles" component={LifeCycles}/>
<Route path="/Binding" component={Binding}/>
<Route path="/ChildOne" component={ChildOne}/>
<Route path="/Parent" component={Parent}/>
<Route path="/ChildTwo" component={ChildTwo}/>

</Switch>
</BrowserRouter>
  
  </div>
  );
 
}
}
//ReactDOM.render(<App/>, <One/>, document.getElementById('root'));
export default App;
