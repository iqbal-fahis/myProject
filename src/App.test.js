import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer'
test("First snapshot testt", ()=>{
  const component=renderer.create(
    <App/>
  
  );
  let tree=component.toJSON();
  expect(tree).toMatchSnapshot();
})