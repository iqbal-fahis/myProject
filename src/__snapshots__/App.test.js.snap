// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First snapshot testt 1`] = `
<div>
  <p
    className="App-header"
  >
    
  </p>
  <p
    className="App-header"
  >
    This is React
  </p>
  <center>
    <button
      onClick={[Function]}
    >
       Increment
    </button>
      
    <button
      onClick={[Function]}
    >
       Decrement
    </button>
  </center>
</div>
`;
